.travis.yml

```diff
@@ -1,13 +1,18 @@
-language: python
+language: minimal
```

Comment:
If using `conda` for testing we don't need the image with `Python`.

Reply:
Good to know.